infrastructure/environments/cloudformation/full/organisation/s3.yaml

```diff
@@ -122,10 +122,6 @@ Resources:
             Id: Backup
             Prefix: ''
             Status: Enabled
-#            SourceSelectionCriteria:
-#              ReplicaModifications:
-#                Status: Disabled
-#                Status: Disabled
```

Comment:
Is this part of the same change?

Reply:
No, different. I noticed a comment that was left over previously and removed it.